deploy/clowder.md

```diff
@@ -219,3 +218,19 @@ but here are some essentials:
   ```
   oc get project -l ephemeral-ns-reserved -L ephemeral-ns-requester-name,ephemeral-ns-reserved
   ```
+
+# Special Notes
+## capacity-allowlist ConfigMap
+The capacity-ingress pod relies on a ConfigMap, called `capacity-allowlist`, found in GitLab.  The most straightforward way to get this config map is to manually add it via command line.  Make sure you're in the appropriate namespace using the `oc project` command, and then you can deploy the ConfigMap to that namespace using the following command:
+
+```bash
+curl https://gitlab.cee.redhat.com/rhsm/swatch-product-allowlist/-/raw/main/templates/capacity-allowlist.yml | oc process -f - | oc apply -f -
+```
+## bonfire "deploy" command and namespace reservation
+If you use `bonfire deploy` without already having a namespace reserved, it will reserve the namespace for you **BUT** if the app doesn't start up in the default amount of time, bonfire will take down/give up the namespace it reserved to begin with.  To get around this, you can manually reserve the namespace, then pass `-n <NAMESPACE>` as an argument when running `bonfire deploy`.
+
+# TL;DR Quickstart Steps
+1. Start bonfire virtual environment
+2. Reserve a namespace
+3. Apply the `capacity-allowlist` configmap
```

Comment:
Should we add a step to import all the other needed stuff?  The `tls`, `pinhead`, and `ingress` secrets for example.  I have a .gitignored file that has all these but we don't have a great way yet to share that stuff around and I had to create the file by combing through Vault to get the secrets.

Reply:
We automated the population of `tls`,`pinhead`, and `ingress` as part of [ENT-4459](https://issues.redhat.com/browse/ENT-4459).